buildSrc/src/main/kotlin/org/jetbrains/SetupMaven.kt

```diff
@@ -1,45 +0,0 @@
-package org.jetbrains
-
-import org.gradle.api.artifacts.Configuration
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.Internal
-import org.gradle.api.tasks.Sync
-import org.gradle.kotlin.dsl.creating
-import org.gradle.kotlin.dsl.dependencies
-import org.gradle.kotlin.dsl.getValue
-import java.io.File
-
-@Suppress("LeakingThis")
-open class SetupMaven : Sync() {
```

Comment:
The properties in this file were moved to an extension in the new `maven-cli-setup.gradle.kts` convention plugin